blocks/api/registration.js

```diff
@@ -68,6 +68,12 @@ export function registerBlockType( name, settings ) {
 		);
 		return;
 	}
+	if ( 'focusable' in settings && ! ( isFunction( settings.focusable ) || isBoolean( settings.focusable ) ) ) {
```

Comment:
Can you explain the purpose of this `focusable` property? It's not clear to me.

We'll also want to include relevant documentation updates in `docs/`

If at all possible, it would be ideal to not impose this implementation detail onto block authors.